src/utils/selectors.ts

```diff
@@ -20,6 +20,13 @@ import {
   makeIsRequestingSelector,
   makeOrderedSelector,
 } from "./firestoreSelectors";
+import {
+  AnimateMapStageOptions,
+  AnimateMapWorldBounds,
+  ReplicatedUser,
+  ReplicatedVenue,
+} from "../store/reducers/AnimateMap";
```

Comment:
These types should all be moved to `types/*` (as called out earlier in this review), not imported from the store/reducer file.